pkg/check/cbt_check.go

```diff
@@ -0,0 +1,77 @@
+package check
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/vmware/govmomi/vim25/mo"
+	"k8s.io/klog/v2"
+)
+
+const CbtProperty = "ctkEnabled"
+
+type CBTResults struct {
+	disabled []string
+	enabled  []string
+}
+
+// CheckCBT this method will check all VMs for a cluster to make sure they have
+// CBT configured the same across them.
+func CheckCBT(ctx *CheckContext) error {
```

Comment:
This check should implement `NodeCheck` interface, so as it can be run parallely on the nodes. Running it as node check will also save some API calls, because we already have node list and vm objects.

Reply:
The reason I did not do NodeCheck  is because I do not know what value to check for.  If cluster has all nodes not defined (not using) ctkEnabled, then that is ok.  If they are using the ctkEnabled, then I can only fail during the NodeCheck and not at the end (meaning each node fails the check immediately).  The first node would set the flag as to whether or not ctkEnabled is TRUE or FALSE.  If FALSE, then all remaining nodes need to be FALSE and similarly with TRUE, all would have to be TRUE to pass the test.  If we had a configuration I could check somewhere to see what the customer wants the VMs of the cluster to be, then I can migrate / modify logic to use NodeCheck versus the cluster wide check.  This is all based on my understanding that the requirement is all nodes need to match.

Reply:
Ok, I'll try to generate a metric and see how that looks.  Thanks for the feedback.